### PR TITLE
Anthrax Mercenary Touch-Ups

### PR DESCRIPTION
**Pull request by SELFHELL**
## About The Pull Request

Slightly expands Anthrax Mercenaries, giving both variants fancy daggers and giving the male ones a new sword.

Also gives males a gorget because the coif looks really ugly.

less compressed

Massive credit to Cre for fixing up my codersprites.

## Why It's Good For The Game

Mostly a consistency thing, seemed pretty weird that one variant of Anthrax merc had fancy weapons while the other didnt.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

### CODE REVIEW

**Review by Optimism333 — APPROVED** [review]
Looks good to me! No balance changes, cool sprites. Easy approval.

**Comment by sanshoom** [comment]
Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.

**Comment by Optimism333** [comment]
> Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.

Hush you

**Comment by SELFHELL** [comment]
its fine I can use the old sprites if you want

> > Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.
> 
> Hush you

File: code/game/objects/items/rogueweapons/melee/knives.dm [hunk 1]
```diff
@@ -169,6 +169,16 @@
 	icon_state = "sdaggeralt"
 	desc = "A dagger of refined steel, and even more refined appearance."
 
+/obj/item/rogueweapon/huntingknife/idagger/steel/dirk
+	name = "fanged dagger"
+	desc = "A dagger modeled after the fang of an anthrax spider."
+	force = 14
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueweapons/melee/knives.dm:175]: No need to redefine force if it inherits from parent

Comment (SELFHELL) [reply code/game/objects/items/rogueweapons/melee/knives.dm:175]: Good point

**Comment by sanshoom** [comment]
The left one is the dagger and the right one is the sword, right?

**Comment by ThePainkiller** [comment]
> Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.

If this is a joke, I hope it is, no need to be mean (I literally cannot read sarcasm on posts)

**Comment by SELFHELL** [comment]
> > Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.
> 
> If this is a joke, I hope it is, no need to be mean (I literally cannot read sarcasm on posts)

it was

**Comment by sanshoom** [comment]
> > Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.
> 
> If this is a joke, I hope it is, no need to be mean (I literally cannot read sarcasm on posts)

yes it is, i just said the opposite of what the other maintainer said 😭

**Comment by Optimism333** [comment]
> > > Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.
> > 
> > 
> > If this is a joke, I hope it is, no need to be mean (I literally cannot read sarcasm on posts)
> 
> yes it is, i just said the opposite of what the other maintainer said 😭

I'm not a maintainer... why do people say I am...

**Comment by sanshoom** [comment]
> > > > Looks terrible to me. There needs to be some balance changes and respriting done here, otherwise I'm not approving it.
> > > 
> > > 
> > > If this is a joke, I hope it is, no need to be mean (I literally cannot read sarcasm on posts)
> > 
> > 
> > yes it is, i just said the opposite of what the other maintainer said 😭
> 
> I'm not a maintainer... why do people say I am... aaaaaaaaaaaa...

code slave, maintainer. same thing really.

**Comment by SELFHELL** [comment]
Should be good to merge whenever,

**Review by ThePainkiller — APPROVED** [review]
Going from an iron shortsword into a **steel cutlass** seems too big of a buff, but if the Zybantu merc can have the super saber and Steppesman gets a steel one, why not? (Grenzel gets a zwei, boltslinger keeps losing were it not for the crossbow)

**Comment by sanshoom** [comment]
i love you btw painkiller

**Pull request merged**